llvm/test/tools/yaml2obj/GOFF/symbol-required.yaml

```diff
@@ -0,0 +1,31 @@
+# RUN: yaml2obj %s | od -v -An -tx1 | FileCheck --ignore-case %s
```

Comment:
This test is named "symbol-required" but it's unclear from the test name what that actually means. Please add a brief comment at the top of the test explaining the purpose of this test. In general, this may be a good idea for all your tests.